src/components/PrintedBallot.tsx

```diff
@@ -193,13 +194,13 @@ interface Props {
 }
 
 const PrintBallot = ({
+  ballotId = randomBase64(),
```

Comment:
I think we should require `ballotId`, otherwise there's a chance we code this incorrectly and generate new ballot IDs on the fly, which is never the right answer.

Reply:
Agreed. There are two other places where this is used. I didn't have the capacity to make these changes last night.